### PR TITLE
🖌️ style: Update Light/Dark UI Themes

### PR DESCRIPTION
**Pull request by berry-13**
## Summary

updated the color scheme to resemble that of ChatGPT and addressed several UI bugs

## added new tooltips

image

image

## Change Type

- [x] Bug fix
- [x] Style update

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.

### CODE REVIEW

**Comment by itzraiss** [comment]
@Berry-13 So with these corrections, my PR #1751 would no longer be necessary. Do you think it would be interesting to add the event of showing the Rename and Delete icons when hovering the mouse?

**Comment by Berry-13** [comment]
> @Berry-13 So with these corrections, my PR #1751 would no longer be necessary. Do you think it would be interesting to add the event of showing the Rename and Delete icons when hovering the mouse?

yeah, it's likely no longer necessary. I also want to add only one button with the three dots, similar to ChatGPT

**Comment by danny-avila** [comment]
Thanks! Btw I might make this part of the next tag/release. What were the bugs fixed?

**Comment by Berry-13** [comment]
> Thanks! Btw I might make this part of the next tag/release. What were the bugs fixed?

sorry, they were not bugs, but rather a result of the wrong style. This includes the style of the conversation (such as the right corners) and the style of the buttons in the preset selection

**Comment by danny-avila** [comment]
I'm thinking of squashing all of your commits to see if it makes the merge simpler. would love to be able to cycle themes from the current (old) with the new since most of is just color palette.

**Comment by peterhoang** [comment]
> To be honest Idk if I like the new dark theme 🤷‍♂️ it’s too dark

Same. Would it be possible to retain the "old" dark theme and not a complete replacement? Perhaps, giving the users the control to choose a "theme"?

**Comment by Berry-13** [comment]
> > To be honest Idk if I like the new dark theme 🤷‍♂️ it’s too dark
> 
> Same. Would it be possible to retain the "old" dark theme and not a complete replacement? Perhaps, giving the users the control to choose a "theme"?

yes, I thought I had already written it, but instead, I was mistaken. Anyway, yes, I will try to add the possibility to change many more themes and maybe even to create them

**Comment by danny-avila** [comment]
Mind fixing this merge conflicts? Looking to test and merge. Also, was it necessary to change every file's tailwind class? I feel it's a missed opportunity to use tailwind themes and match the classes where applicable.

**Comment by danny-avila** [comment]
see https://github.com/L-Blondy/tw-colors

File: client/src/components/Chat/CreationHeader.tsx
Comment (danny-avila) [inline client/src/components/Chat/CreationHeader.tsx:1]: this file should not be added back to the project

File: client/src/components/Chat/CreationPanel.tsx
Comment (danny-avila) [inline client/src/components/Chat/CreationPanel.tsx:1]: this file should not be added back to the project

**Comment by Berry-13** [comment]
> I feel it's a missed opportunity to use tailwind themes and match the classes where applicable.

Sure, I can make a new PR about that. This is just a UI update. Thanks for sharing the "tw-colors". I'll integrate it in a next PR, since it will take a lot more time to merge this then

**Comment by danny-avila** [comment]
> > I feel it's a missed opportunity to use tailwind themes and match the classes where applicable.
> 
> Sure, I can make a new PR about that. This is just a UI update. Thanks for sharing the "tw-colors". I'll integrate it in a next PR, since it will take a lot more time to merge this then

I may accept this PR first but I believe it will be harder to integrate the old style that is being replaced once it's gone. You can always use this PR for reference at least.

**Comment by danny-avila** [comment]
@Berry-13 bg-gray-800 was much darker for us (which is what I think ChatGPT had it originally as) so I updated since ChatGPT was noticably lighter which I liked. Also updated styling some other areas.

**Comment by Berry-13** [comment]
> @Berry-13 bg-gray-800 was much darker for us (which is what I think ChatGPT had it originally as) so I updated since ChatGPT was noticably lighter which I liked. Also updated styling some other areas.

ok, thanks

**Review by danny-avila — APPROVED** [review]

**Comment by olinorwell** [comment]
Feedback from my users (mostly family and inside my office) is that the old style from last year up till a few weeks ago was much loved - the new style is too dark and although it might match OpenAI better - the general consensus is that the old one is significantly better to look at. It would be hugely popular if there was a way to keep the old theme, or at least allow the background colour to be set. I thought about trying to hack a solution but thought I'd mention it as it seems like others agree.

**Comment by Berry-13** [comment]
> Feedback from my users (mostly family and inside my office) is that the old style from last year up till a few weeks ago was much loved - the new style is too dark and although it might match OpenAI better - the general consensus is that the old one is significantly better to look at. It would be hugely popular if there was a way to keep the old theme, or at least allow the background colour to be set. I thought about trying to hack a solution but thought I'd mention it as it seems like others agree.

sure, I'm already working on a theme option. Right now, my goal is to add the old style and also give the user the option to customize themes directly from the UI

**Pull request merged**